### ISSUE TITLE
Fix support for multipolygons

### ISSUE THREAD

**Issue by springmeyer** [maplibre/maplibre-tile-spec#112]
Currently we cannot upgrade to https://github.com/ElectronicChartCentre > 1.3.4 due to https://github.com/ElectronicChartCentre/java-vector-tile/issues/18. Otherwise when multi polygons are encoded in MVTs properly (rather than being split) our code cannot cope, hitting errors like:

```
MltDecoderTest > decodeMlTile_Z2() FAILED
    java.lang.IllegalArgumentException: holes must not contain null elements
        at org.locationtech.jts.geom.Polygon.<init>(Polygon.java:122)
        at org.locationtech.jts.geom.GeometryFactory.createPolygon(GeometryFactory.java:467)
        at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:202)
        at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:67)
        at com.mlt.decoder.MltDecoderTest.testTile(MltDecoderTest.java:82)
        at com.mlt.decoder.MltDecoderTest.decodeMlTile_Z2(MltDecoderTest.java:31)

MltDecoderTest > decodeMlTile_Z4() FAILED
    java.lang.IllegalArgumentException: holes must not contain null elements
        at org.locationtech.jts.geom.Polygon.<init>(Polygon.java:122)
        at org.locationtech.jts.geom.GeometryFactory.createPolygon(GeometryFactory.java:467)
        at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:202)
        at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:67)
        at com.mlt.decoder.MltDecoderTest.testTile(MltDecoderTest.java:82)
        at com.mlt.decoder.MltDecoderTest.decodeMlTile_Z4(MltDecoderTest.java:37)

MltDecoderTest > decodeMlTile_Z5() FAILED
    java.lang.IllegalArgumentException: geometries must not contain null elements
        at org.locationtech.jts.geom.GeometryCollection.<init>(GeometryCollection.java:54)
        at org.locationtech.jts.geom.MultiPolygon.<init>(MultiPolygon.java:64)
        at org.locationtech.jts.geom.GeometryFactory.createMultiPolygon(GeometryFactory.java:346)
        at com.mlt.decoder.GeometryDecoder.decodeGeometry(GeometryDecoder.java:203)
        at com.mlt.decoder.MltDecoder.decodeMlTile(MltDecoder.java:67)
        at com.mlt.decoder.MltDecoderTest.testTile(MltDecoderTest.java:82)
        at com.mlt.decoder.MltDecoderTest.decodeMlTile_Z5(MltDecoderTest.java:46)
```

I appears that we are able to encode MLT tiles (correctly?) but then we cannot decode our own MLT in the java decoder.

**Comment by springmeyer** [comment]
@mactrem is this something you could work on fixing?

**Comment by mactrem** [comment]
@springmeyer yes, i will have a look at it

**Comment by springmeyer** [comment]
Thanks @mactrem! I'll be curious to know if you have a sense of whether mlt encoding is likely correct and this is just a decoding issue.

**Comment by springmeyer** [comment]
@mactrem note that https://github.com/maplibre/maplibre-tile-spec/pull/111 upgraded from `no.ecc.vectortile:java-vector-tile:1.3.1` to `no.ecc.vectortile:java-vector-tile:1.3.3`, just not further (due to this issue).

**Comment by mactrem** [comment]
Fixed in #79

**Comment by mactrem** [comment]
Fixed in https://github.com/maplibre/maplibre-tile-spec/pull/79
